### PR TITLE
Bug 1122080 - Test that the play icon is in the status bar.

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[hfiguiere Hubert Figuière (hfiguiere)](https://github.com/hfiguiere) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b14945b901b1)

**Comment by try-server-hook** [comment]
[hfiguiere Hubert Figuière (hfiguiere)](https://github.com/hfiguiere) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=96077c8ed47e)

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2e5432685e27314662d735a24b08e99662f29e62

File: apps/music/test/marionette/Player_test.js [hunk 1]
```diff
@@ -131,6 +136,36 @@ marionette('Music player tests', function() {
...
     });
+
+    test('Check the play icon is in the status bar', function() {
+      music.launch();
+      music.waitForFirstTile();
+      music.switchToSongsView();
+
+      // check the status bar for the hidden play icon
+      client.switchToFrame();
+      client.findElement('#statusbar-maximized');
```
Comment (davidflanagan) [inline apps/music/test/marionette/Player_test.js]: I don't understand what the findElement is doing here and below. If you really need it, maybe add a comment explaining why you're finding this element and discarding the result. Same comment below.

File: apps/music/test/marionette/Player_test.js [hunk 2]
```diff
@@ -131,6 +136,36 @@ marionette('Music player tests', function() {
       music.tapHeaderActionButton();
       music.checkPlayerIconShown(true);
     });
+
+    test('Check the play icon is in the status bar', function() {
```
Comment (davidflanagan) [inline apps/music/test/marionette/Player_test.js]: This seems different enough from the other tests in this suite that you could consider creating a new suite for this test.  I don't feel strongly about it, though.

File: apps/music/test/marionette/Player_test.js (hunk 1 above)
Comment (hfiguiere) [inline apps/music/test/marionette/Player_test.js]: I'm not sure why I left them here, but they are gone now. Both. Good catch.

File: apps/music/test/marionette/Player_test.js (hunk 2 above)
Comment (hfiguiere) [inline apps/music/test/marionette/Player_test.js]: Make sense. I created a new suite. Move the new hunk above into it.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9a53e2b028408562a5daf2954750496da40d401e

**Pull request merged**